.github/CODEOWNERS

```diff
@@ -256,6 +256,15 @@
 
 /theories/Vectors/        @herbelin
 
+########## Dune  ##########
+
+/Makefile.dune         @ejgallego
+/tools/coq_dune*       @ejgallego
+/dune*                 @ejgallego
+/coq.opam              @ejgallego
+/ide/coqide.opam       @ejgallego
+# Secondary maintainer @Zimmi48
```

Comment:
Should we have a dune team from the start instead?

Reply:
As you prefer folks.

Reply:
In this case, I see a strong primary / secondary knowledge difference so I would say no. Let's expand the use of teams in domains where the knowledge is better shared first.